src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.tsx

```diff
@@ -157,8 +189,13 @@ export const RespondentDataReview = () => {
     handleGetSubmitDates(date);
   };
 
+  const handleResponseDateChange = async (date?: Date | null) => {
```

Comment:
looks like the function operations are synchronous, including the call to `handleGetActivities`, is async needed here?